integration-libs/opf/checkout/components/opf-checkout-payments/opf-checkout-payments.component.html

```diff
@@ -31,12 +31,21 @@ <h3 *ngIf="isHeadingDisplayed">
                   class="form-check-input"
                   role="radio"
                   type="radio"
-                  [attr.aria-checked]="configuration.id === selectedPaymentId"
+                  [attr.aria-checked]="
+                    (paymentDisabled$ | async) &&
```

Comment:
We can use single async pipe instead of several ones. This reduces subscriptions and keeps the template readable.
Example:
```
<ng-container *ngIf="paymentDisabled$ | async as isPaymentDisabled">
```
then `isPaymentDisabled` can be used everywhere under specific block/element.